README.md

```diff
@@ -362,5 +362,6 @@ This repository is open source and available under the MIT license. For more inf
 [reference.AtomeComponent]:  https://adyen.github.io/adyen-ios/5.0.0/documentation/adyen/atomecomponent
 [reference.UPIComponent]:  https://adyen.github.io/adyen-ios/5.0.0/documentation/adyen/upiComponent
 [reference.QRCodeActionComponent]:  https://adyen.github.io/adyen-ios/5.0.0/documentation/adyen/qrCodeActionComponent
+[reference.CashAppPayComponent]:  https://adyen.github.io/adyen-ios/5.0.0/documentation/adyen/cashAppPayComponent
```

Comment:
This line doesn't do anything on it's own.

Reply:
Yeah doc isn't created before release, let's discuss this too.

Reply:
No, I mean the Markdown `[some_name]:  https://some.url.com` is just a reference. 
It must be used somewhere, like [here](https://github.com/Adyen/adyen-ios/blob/5bb1f9fd5b60c2006ee5b12acd06acbce479ac12/README.md?plain=1#L269)

Reply:
I think we need to add this in `Available Components` as well. I did for UPI and QRCode.